src/ipfsd-daemon.js

```diff
@@ -239,7 +239,24 @@ class Daemon {
       return this
     }
 
-    await this.api.stop()
+    let killTimeout
+    let killed = false
+    if (this.opts.forceKill !== false) {
+      killTimeout = setTimeout(() => {
+        // eslint-disable-next-line no-console
+        console.error(new Error(`Timeout stopping ${this.opts.type} node. Process ${this.subprocess.pid} will be force killed now.`))
+        killed = true
+        this.subprocess.kill('SIGKILL')
+      }, this.opts.forceKillTimeout || 10000)
```

Comment:
can you add a default to 10000 here https://github.com/ipfs/js-ipfsd-ctl/blob/3dae27670fdebe0edd6e1e26ddf5df4c274b62bf/src/factory.js#L20 and in the docs